lib/commands/createcache.js

```diff
@@ -17,6 +17,18 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Cache Resource',
     shortOption: 'z',
     required: true
+  },
+  description: {
+    name: 'Cache description',
+    required: false
+  },
+  cacheExpiryByDate:{
+    name: 'Cache expiration by date (mm-dd-yyyy)',
+    required: false
+  },
+  cacheExpiryInSecs:{
```

Comment:
Missing space after `:`.

Reply:
PR updated to fix code formatting issue.